android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -197,6 +197,12 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
             "change_name"->{
                 changeName(call)
             }
+            "hls_start_streaming"->{
+                hlsStreaming(call)
```

Comment:
`result` should be sent. result(nil) if success, or exception dictionary if an error occurs